src/components/LabeledRangeSlider.tsx

```diff
@@ -105,7 +106,7 @@ const SliderLabel = styled.p<{ $disabled?: boolean }>`
  * separately from the min and max value bounds and acts as a suggested range.
  */
 export default function LabeledRangeSlider(inputProps: LabeledRangeSliderProps): ReactElement {
-  const props = { ...defaultProps, ...inputProps } as Required<LabeledRangeSliderProps>;
+  const props = { ...defaultProps, ...excludeUndefinedValues(inputProps) } as Required<LabeledRangeSliderProps>;
```

Comment:
question: does this imply that you can't force a value to be explicitly undefined through inputProps?

Reply:
Good question! Yes, at least for this instance. I was having an issue where the default values for `minSliderBounds` and `maxSliderBounds` were getting overridden by `undefined` input props.

Reply:
I could probably add in a manual check for those two properties, but it seemed like that might cause problems elsewhere too.